src/olympia/reviewers/decorators.py

```diff
@@ -117,3 +117,22 @@ def wrapper(request, *args, **kw):
             return f(request, *args, **kw)
         raise PermissionDenied
     return wrapper
+
+
+def any_reviewer_or_moderator_required(f):
+    """Like @any_reviewer_required, but allows users with Ratings:Moderate
```

Comment:
This doesn't seem very DRY :)  Is refactoring practical?

Reply:
I tried... just could not find the motivation. It basically means writing decorators that accept arguments, and I find those always extra confusing and harder to debug...

Reply:
Fair.  I don't care that much.